src/core/client/stream/tabs/comments/containers/ReportButtonContainer.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { graphql } from "react-relay";
+
+import { withFragmentContainer } from "talk-framework/lib/relay";
+import { ReportButtonContainer_comment as CommentData } from "talk-stream/__generated__/ReportButtonContainer_comment.graphql";
+import { ReportButtonContainer_me as MeData } from "talk-stream/__generated__/ReportButtonContainer_me.graphql";
+
+import {
+  ReportButton,
+  ReportButtonWithPopover,
+} from "../components/ReportButton";
+
+import {
+  ShowAuthPopupMutation,
+  withShowAuthPopupMutation,
+} from "talk-stream/mutations";
+
+interface ReportButtonContainerProps {
+  comment: CommentData;
+  me: MeData | null;
+  showAuthPopup: ShowAuthPopupMutation;
+}
+
+class ReportButtonContainer extends React.Component<
+  ReportButtonContainerProps
+> {
+  private get loggedIn() {
+    return this.props.me;
```

Comment:
It seems like this is used frequently.. Is there some way to abstract this? Even adding a note that it should be extracted would be fine.

Reply:
I don't think there is something to be extracted here. How would that look like?